app/master/subjob.py

```diff
@@ -103,29 +98,6 @@ def atomic_commands(self):
         job_command = self.job_config.command
         return ['{} {}'.format(atom.command_string, job_command) for atom in self._atoms]
 
-    def get_atom_artifact_name(self, atom_id):
```

Comment:
I think static is only bad when used incorrectly, and otherwise is better than the alternative. If you can give me a good reason why this is not a good use case for it, I'm happy to change it.